src/Resources/laravel-debugbar.css

```diff
@@ -345,14 +356,26 @@ ul.phpdebugbar-widgets-timeline table.phpdebugbar-widgets-params {
     font-size: 11px;
 }
 
+ul.phpdebugbar-widgets-timeline table.phpdebugbar-widgets-params td {
+    border-left: none ;
+    border-right: none;
+}
+
+ul.phpdebugbar-widgets-timeline table.phpdebugbar-widgets-params td:first-child {
+    padding: 0 10px;
+}
+
 div.phpdebugbar-widgets-messages div.phpdebugbar-widgets-toolbar {
-    width: calc(100% - 20px);
```

Comment:
This is a regression, 
the `-20px` was there to prevent the toolbar from obstructing the vertical scrollbar, on all the tabs extending messages

![image](https://github.com/barryvdh/laravel-debugbar/assets/4933954/44f93b5b-8cb9-425e-ba81-e59ab81ff637)

![image](https://github.com/barryvdh/laravel-debugbar/assets/4933954/07e06b1f-22db-4d37-9e25-5b5cddfd9147)

Reply:
The same goes for the close button then? I expect you have this issue as well with the base package? http://phpdebugbar.com. On Mac the scrollbars hide or are half translucent when scrolling so I didn't encountered this. Is this IE or something?